cob_cartesian_controller/src/cartesian_controller.cpp

```diff
@@ -177,6 +174,9 @@ bool CartesianController::posePathBroadcaster(const geometry_msgs::PoseArray& ca
 
         tf_broadcaster_.sendTransform(tf::StampedTransform(transform, ros::Time::now(), cartesian_path.header.frame_id, target_frame_));
 
+        // Activate Tracking
+        startTracking(); // Somehow the behavior changed.. we used to call this one before the first pose was published but this doesn't work anymore.
+
```

Comment:
with this you are calling the start_tracking service for every pose on the path. Is this really intended? You probably just have to take care that the transform specified in the path is available before calling tracking service.